### ISSUE TITLE
when i try to using sam demo,i get this problem ,is that the version problem? how should i do for this issues?

### ISSUE THREAD

**Issue by Vincent630** [kornia/kornia#2432]
### Describe the bug

when i try to using sam demo,i get this problem ,is that the version problem? how should i do for this issues?

### Reproduction steps

```bash
none
```

### Expected behavior

import this module and test SAM 

### Environment

```shell
python 3.9,kornia 0.6.12,cuda 11.3,pytorch1.12
```

**Comment by edgarriba** [comment]
@Vincent630 what´s the error issue you get ?

**Comment by Vincent630** [comment]
ModuleNotFoundError: No module named 'kornia.contrib.visual_prompter'

**Comment by edgarriba** [comment]
I do you do `from kornia.contrib.visual_prompter import VisualPrompter` , it doesn’t work ?

**Comment by Vincent630** [comment]
yes,and i can use kornia geometry to do something about 3d  computer vision
![image](https://github.com/kornia/kornia/assets/30228037/58268a51-faa5-460e-8380-e8069b3dba1b)
>>> kornia.__version__
'0.6.12'
>>> import kornia.contrib.
kornia.contrib.ClassificationHead(      kornia.contrib.FaceKeypoint(            kornia.contrib.compute_padding(         kornia.contrib.face_detection
kornia.contrib.CombineTensorPatches(    kornia.contrib.ImageStitcher(           kornia.contrib.connected_components(    kornia.contrib.histogram_matching(
kornia.contrib.DistanceTransform(       kornia.contrib.Lambda(                  kornia.contrib.diamond_square(          kornia.contrib.image_stitching
kornia.contrib.EdgeDetector(            kornia.contrib.MobileViT(               kornia.contrib.distance_transform(      kornia.contrib.interp(
kornia.contrib.ExtractTensorPatches(    kornia.contrib.VisionTransformer(       kornia.contrib.edge_detection           kornia.contrib.lambda_module
kornia.contrib.FaceDetector(            kornia.contrib.classification           kornia.contrib.extract_patches          kornia.contrib.vit
kornia.contrib.FaceDetectorResult(      kornia.contrib.combine_tensor_patches(  kornia.contrib.extract_tensor_patches(  kornia.contrib.vit_mobile

**Comment by Vincent630** [comment]
i just samply pip install kornia followed offical tutorial

**Comment by edgarriba** [comment]
@Vincent630 i confirm that's bug. I remember now @johnnv1 mentioned something about an issue with some circular imports. We'll try to fix asap

**Comment by Vincent630** [comment]
ok, thank you so much, i will try again when it fixed.

**Comment by gau-nernst** [comment]
v0.6.12 doesn't have visual prompter yet

https://github.com/kornia/kornia/tree/v0.6.12/kornia/contrib

I think you can either use the latest kornia from git or use ImagePrompter instead.
